js/components/Message/Types/JSObject/__tests__/Date.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
As we discussed, let's move all of the tests into a single folder somewhere so it's easier to port into tree (maybe in the root level test directory)